Web/react-spa/src/features/stores/storeAPI.ts

```diff
@@ -0,0 +1,93 @@
+import authService from "app/authService";
+import Store from "./store";
+
+const URL: string = process.env.REACT_APP_API_URL + '/api/s/store'
+
+async function fetchStores() {
+    const responce = await fetch(URL, {
```

Comment:
a common practice is to declate backend.ts and inside of it to declare a set of methods like 
sendRequest({url, method, headers, body})

so that you can simply pass request arguments and simplify boilerplate code with promise resolve